api/src/main/resources/jakarta/data/messages/Messages.properties

```diff
@@ -0,0 +1,29 @@
+#
+# Copyright (c) 2025 Contributors to the Eclipse Foundation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+001.arg.required=The {0} argument is required.
+002.no.elements=The {0} list cannot be empty.
+003.null.element=The {0} list cannot not have a null element.
+004.arg.negative=The {0} argument cannot be negative.
+005.zero.not.allowed=The {0} argument cannot be 0.
+006.zero.size.key=The key must consist of at least one element.
+007.wildcard.conflict=Cannot use the same character ({0}) for both wildcards.
+008.escape.conflict=Cannot use the same character ({0}) for both a wildcard \
+ and escape character.
+009.unknown.number.type=Unexpected subtype of Number: {0}.
+010.unknown.total=A total count of elements is not available.
```

Comment:
Personally, I would leave off the periods.

Reply:
> Personally, I would leave off the periods.

Sure, that's fine. Some of the messages were inconsistent and I had went with the approach that I'm used to from our tech writers asking us to do for our own products. I checked with other Jakarta EE specs do for messages and it looks like Jakarta Servlet leaves off the period, so we can align with that.  It's easy to switch now with all of the messages in one place.